lib/crypto/src/crypto.erl

```diff
@@ -1816,6 +1818,53 @@ aead_tag_len(sm4_ccm) -> 16;
 aead_tag_len(_) ->
     error({badarg, "Not an AEAD cipher"}).
 
+
+
+-doc("""
+Initializes AEAD cipher.
+
+Similar to 'crypto_one_time_aead/7' but only does the initialization part,
+returns a handle to be used with 'crypto_one_time_aead/4'.
+
+""").
```

Comment:
Maybe say something about the possibility to use the same initialized handle several times.